tests/unit/mainwindow/test_prompt.py

```diff
@@ -130,3 +131,22 @@ def test_root_path(self, get_prompt):
         """With / as path, show root contents."""
         prompt = get_prompt('/')
         assert prompt._file_model.rootPath() == '/'
+ 
+    def test_expand_user(self, qtbot, get_prompt, monkeypatch, tmp_path):
+        for directory in ['bar', 'bat', 'foo']:
+            (tmp_path / directory).mkdir()
+
+        homedir = str(tmp_path)
+        monkeypatch.setenv('HOME', homedir)  # POSIX
+        monkeypatch.setenv('USERPROFILE', homedir)  # Windows
+        assert str(pathlib.Path.home()) == homedir
```

Comment:
If you want to write a second test function function for this test (which probably makes sense for the replace count test), you could move this to a fixture.